src/macaron/slsa_analyzer/checks/build_as_code_check.py

```diff
@@ -117,14 +118,16 @@ def _has_deploy_command(self, commands: list[list[str]], build_tool: BaseBuildTo
                 # If there are no deploy args for this build tool, accept as deploy command.
                 # TODO: Support multi-argument build keywords, issue #493.
                 if not build_tool.deploy_arg:
-                    logger.info("No deploy arguments required. Accept %s as deploy command.", str(com))
-                    return str(com)
+                    com_str = json.dumps(com)
```

Comment:
There is a [warning](https://docs.python.org/3/library/json.html#json.dumps) on `json` documentation page about parsing untrusted input, but unfortunately doesn't give an advice on sanitizing the value.

Reply:
Based on our offline discussion we should have a consistent solution throughout Macaron, such as putting a limit on the size of inputs. We can work on this issue separately.